src/config/utils.ts

```diff
@@ -113,3 +113,12 @@ function transformLoglevelToInt(loglevel: Loglevel): number {
       return 1;
   }
 }
+
+export function parseOptionalInt(
+  value: string | undefined,
```

Comment:
```suggestion
  value?: string,
```
Maybe use this to be consistent?

Reply:
fixed